CHANGELOG.md

```diff
@@ -1,8 +1,487 @@
 # go-ipfs changelog
 
-## 0.4.20-rc2 2019-04-11
+## 0.4.20 2019-04-16
 
-TODO
+We're happy to release go-ipfs 0.4.20. This release includes some critical
+performance and stability fixes so all users should upgrade ASAP.
+
+This is also the first release to use go modules instead of GX. While GX has
+been a great way to dogfood an IPFS-based package manager, building and
+maintaining a custom package manager is a _lot_ of work and we haven't been able
+to dedicate enough time to bring the user experience of gx to an acceptable
+level. You can read [#5850](https://github.com/ipfs/go-ipfs/issues/5850) for
+some discussion on this matter.
+
+### Docker
+
+As of this release, it's now much easier to run arbitrary IPFS commands within
+the docker container:
+
+```bash
+> docker run ipfs/go-ipfs config profile apply server # apply the server profile
```

Comment:
I think some users not familiar with Docker may just  copy&paste commands and get surprised that container started by the second command does not include changes applied by the first one.

A safer example would be to use the same container in both, e.g:

```console
$ docker run --name my-container ipfs/go-ipfs:v0.4.20 config profile apply server # apply the server profile
$ docker start my-container # start the daemon
```

Reply:
As someone not familiar with docker, thanks...